app/templates/main/health.html

```diff
@@ -0,0 +1,16 @@
+{% import 'macros/page_macros.html' as page %}
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui text container">
+        <h1 class="ui header">
+        	Health Insurance
+        	{% if current_user.is_admin() %}
+        		<div class="sub header">This is what the user will see on the health page</div>

```

Comment:
Do you mean under ui header because I do have it as health insurance or do you mean the file itself.